src/services/authors.ts

```diff
@@ -2,75 +2,75 @@ export const authors = [
+    { name: "Ralf Schumacher", img: "https://www.f1-photo.com/Ressources/Gallery/1997/Austria/RalfSchumacher_1997_Austria_01_PHC.jpg", id: "ralf-schumacher" },
+    { name: "Ayrton Senna", img: "https://www.f1-photo.com/Ressources/Gallery/1985/Belgium/Senna_1985_Belgium_04_PHC.jpg", id: "senna" },
+    { name: "Günther Steiner", img: "https://cdn-2.motorsport.com/images/mgl/0qX74dO6/s8/gunther-steiner.jpg", id: "steiner" },
+    { name: "Jackie Stewart", img: "https://www.f1-photo.com/Ressources/Gallery/1970/Spain/Stewart_1970_Spain_02_BC.jpg", id: "stewart" },
+    { name: "Lance Stroll", img: "https://cdn-8.motorsport.com/images/amp/6gp5XL70/s1000/lance-stroll-aston-martin-f1-t.webp", id: "stroll" },
+    { name: "John Surtees", img: "https://www.f1-photo.com/Ressources/Gallery/1964/Holland/Surtees_1964_Holland_01_BC.jpg", id: "surtees" },
+    { name: "Jarno Trulli", img: "https://www.f1-photo.com/Ressources/Gallery/2008/Japan/Trulli_2008_Japan_03_PHC.jpg", id: "trulli" },
     { name: "Yuki Tsunoda", img: "https://www.formula1.com/content/fom-website/en/drivers/yuki-tsunoda/jcr:content/image.img.1920.medium.jpg/1669044222425.jpg", id: "tsunoda" },
-    { name: "Jos Verstappen", img: "", id: "jos-verstappen" },
-    { name: "Max Verstappen", img: "https://www.formula1.com/content/fom-website/en/drivers/max-verstappen/jcr:content/image.img.1024.medium.jpg/1670841844162.jpg", id: "verstappen" },
-    { name: "Sebastian Vettel", img: "https://media.gq.com.mx/photos/62e28f535a8c1e60ded0d872/1:1/w_1999,h_1999,c_limit/Sebastian-Vettel-en-Aston-Martin.jpg", id: "vettel" },
-    { name: "Gilles Villeneuve", img: "", id: "gilles-villeneuve" },
-    { name: "Jacques Villeneuve", img: "", id: "jacques-villeneuve" },
-    { name: "Mark Webber", img: "", id: "webber" },
-    { name: "Frank Williams", img: "", id: "williams" },
-    { name: "Susie Wolff", img: "", id: "susie-wolff" },
+    { name: "Jos Verstappen", img: "https://cdn-9.motorsport.com/images/mgl/6VRQWkw6/s8/yuki-tsunoda-visa-cash-app-rb-.jpg", id: "jos-verstappen" },
+    { name: "Max Verstappen", img: "https://cdn-4.motorsport.com/images/mgl/YEQxVmpY/s8/max-verstappen-red-bull-racing.jpg", id: "verstappen" },
+    { name: "Sebastian Vettel", img: "https://www.f1-photo.com/Ressources/Gallery/2012/Spain/Vettel_2012_Spain_02_PHC.jpg", id: "vettel" },
+    { name: "Gilles Villeneuve", img: "https://www.f1-photo.com/Ressources/Gallery/1977/England/Villeneuve_1977_England_01_PHC.jpg", id: "gilles-villeneuve" },
+    { name: "Jacques Villeneuve", img: "https://www.f1-photo.com/Ressources/Gallery/2003/Germany/Villeneuve_2003_Germany_01_PHC.jpg", id: "jacques-villeneuve" },
+    { name: "Mark Webber", img: "https://www.f1-photo.com/Photo/Webber_2012_Monaco_10_PHC.jpg", id: "webber" },
+    { name: "Frank Williams", img: "https://www.f1-photo.com/Photo/Rosberg-Williams_1984_Monaco_01_PHC.jpg", id: "williams" },
```

Comment:
Please put the single photo URL, not the webpage URL where it's the photo, sorry.